csrc/transformer/inference/csrc/transform.cu

```diff
@@ -683,7 +683,7 @@ void launch_transform4d_0213<float>(float* out,
 }
 
 template <typename T>
-void launch_transform4d_0213<T>(T* out,
+void launch_transform4d_0213(T* out,
```

Comment:
@tjruwase https://github.com/microsoft/DeepSpeed/pull/3092 introduced this change. It seems like most of the inference UTs skipped in the CUDA checks? https://github.com/microsoft/DeepSpeed/actions/runs/4773168080/jobs/8486085194?pr=3092#step:8:29
Does this code work for CUDA?

Reply:
cc @jeffra @mrwyattii